.devhubignore

```diff
@@ -0,0 +1,24 @@
+/.babelrc
+/.devhubignore
+/.dockerignore
+/.editorconfig
+/.env
+/.eslintignore
+/.eslintrc
+.git/
+/.gitignore
+/.prettierrc
+.s2i/
+.vscode/
+/Caddyfile
+/Dockerfile
+/Jenkinsfile
+/LICENSE
+README.md
+/devhub.md
```

Comment:
this should be removed from the ignore file otherwise the devhub will ignore it and not generate a card for that markdown file

Reply:
Fixed.